content/en/platform/corda/4.10/enterprise/node/operating/monitoring-and-logging/opentelemetry.md

```diff
@@ -0,0 +1,111 @@
+
+{{< note >}}
+The two types of telemetry are completely independent. You can have both, one, or none of them enabled.
+{{< /note >}}
+
+
+## Corda OpenTelemetry integration
+
+The Corda OpenTelemetry component links at compile time with the OpenTelemetry API. In order to produce spans, there is a runtime dependency on the OpenTelemetry SDK. 
+
+There are two ways to make the OpenTelemetry SDK available to the Corda node:
+
+* Include the `corda-opentelemetry-driver` into the drivers directory of the Corda node.
+* Start the Corda node with an OpenTelemetry Java agent installed.
+
+## Using the OpenTelemetry driver
+
+To use the OpenTelemetry driver, copy `corda-opentelemetry-driver-%VERSION%.jar` into the drivers directory of the Corda node. This driver can be downloaded from Artifactory and is included in the release pack.
+
+## Using the OpenTelemetry Java agent
+
+To use the OpenTelemetry Java agent:
+
+1. Download the agent from [GitHub](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases). The file you need is `opentelemetry-javaagent.jar`.
+2. Start the Corda node with the following example command line: `java -Dcapsule.jvm.args=“-javaagent:/PATH-TO-OT-JAVAAGENT/opentelemetry-javaagent.jar -Dotel.service.name=YOUR-SERVICE-NAME” -jar corda.jar`.
+3. Replace `PATH-TO-OT-JAVAAGENT` with the full path to where you placed the OpenTelemetry Java agent.
+4. Replace `YOUR-SERVICE-NAME` with the service name you would like the Corda node to be identified as.
+
+{{< note >}}
+This method of linking the SDK with the Corda node enables OpenTelemetry automated instrumentation, which means some other third-party libraries included with Corda will also generate spans. See the [OpenTelemetry](https://opentelemetry.io/docs/concepts/instrumenting-library/) website for details on which libraries are enabled.
+{{< /note >}}
+
+
+## Setting up OpenTelemetry
+
+Corda has been tested with the OpenTelemetry collector and a Jaeger backend. Refer to the [OpenTelemetry](https://opentelemetry.io/docs/collector/) and [Jaeger](https://www.jaegertracing.io/) websites for details on how to setup a collector and Jaeger backend.
+
+Corda OpenTelemetry can also be enabled or disabled via a node configuration parameter, such as `telementry.openTelemetryEnabled = true`.
+
+If this setting is true and the OpenTelemetry SDK has been linked to Corda as described above, the node will generate spans. If this setting is false, the node won't generate spans, even if the node is linked with the SDK.
+
+## Client API
+
+OpenTelemetry span generation is also incorporated into the RPC client API. If you want your Corda client code to generate spans, you need to include the OpenTelemetry SDK as a dependency. The methods described above only apply to the Corda node. 
+
+<!-- (TODO: Give example of dependencies needed). -->
```

Comment:
This answers the TODO:

You can link to the opentelemetry SDK from a client project in two ways. By compiling against the open telemetry driver which includes the open telemetry SDK. Or by running the client with the open telemetry java agent attached.

To use the open telemetry driver add the following dependency to the build.gradle file of the client.

dependencies { 
...
compile "net.corda:corda-opentelemetry-driver:$corda_release_version"
}

Alternatively you can attach your client to the open telemetry java agent via the following command line. 

java -javaagent:%PATH-TO-OT-JAVAAGENT%/opentelemetry-javaagent.jar -Dotel.service.name=YOUR-SERVICE-NAME -jar ./your-client.jar